src/components/popover/popover.css

```diff
@@ -0,0 +1,32 @@
+@import '../../styles/colors-v2.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography-v2.css';
+
+.close-button {
+  composes: button from '../icon-button/icon-button.css';
+  height: 12px;
+  margin-left: var(--spacing-large);
+  padding: var(--spacing-small);
+  padding-top: 5px;
+  width: 12px;
+}
+
+.container {
+  background-color: var(--white);
+  border: 1px solid var(--mds-grey-12);
+  box-shadow: var(--spacing-small) var(--spacing-small) var(--spacing-small) var(--mds-grey-8);
+  padding: var(--spacing-large);
+  position: fixed;
```

Comment:
Just to track this: we decided against `fixed` in favor of `absolute` so the popover element scrolls with the page.